Calculator/Calculator/Controller/CalculatorViewController.swift

```diff
@@ -4,131 +4,81 @@ final class CalculateViewController: UIViewController {
     @IBOutlet weak var enteringNumberLabel: UILabel!
     @IBOutlet weak var enteringOperatorLabel: UILabel!
     @IBOutlet weak var calculatorScrollView: UIScrollView!
-    @IBOutlet weak var calculatorStackView: UIStackView!
+    @IBOutlet weak var calculatorStackView: CalculatorStackView!
     
-    private var enteringNumber: String = Sign.empty {
-        didSet {
-            enteringNumberLabel.text = calculatorChecker.hasCurrentInput(enteringNumber) ? enteringNumber : Sign.zero
-        }
-    }
-    private var calculationExpression: String = Sign.empty
-    
-    private let calculatorChecker = CalculatorChecker()
+    private var calculatorChecker: CalculatorChecker?
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        calculatorChecker = CalculatorChecker(updateClosure: { enteringNumber in
+            self.enteringNumberLabel.text = enteringNumber == Sign.empty ? Sign.zero : enteringNumber
```

Comment:
한 눈에 보고 어떤 코드인지 바로 파악하기 어려워보여요.
조건문으로 분리하거나 괄호로 분리해서 표시하는게 좋겠습니다!
```suggestion
            self.enteringNumberLabel.text = (enteringNumber == Sign.empty) ? Sign.zero : enteringNumber
```